### PR TITLE
Node.js - fuzzy search in next release

### PR DESCRIPTION
**Pull request by johannes-vogel**

### CODE REVIEW

File: guides/providing-services.md [hunk 1]
```diff
@@ -420,14 +420,30 @@ You can explicitly annotate calculated elements to make them searchable, even th
 
 Fuzzy search is a fault-tolerant search feature of SAP HANA Cloud, which returns records even if the search term contains additional characters, is missing characters, or has typographical errors.
 
-If you run CAP Java in [`HEX` optimization mode](../java/cqn-services/persistence-services#sql-optimization-mode) on SAP HANA Cloud, you can enable fuzzy search in the *application.yaml* and configure the default fuzziness in the range [0.0, 1.0]. The value 1.0 enforces exact search. The default fuzziness is 0.8.
+If you run in [`HEX` optimization mode](../java/cqn-services/persistence-services#sql-optimization-mode) on SAP HANA Cloud, you can enable fuzzy search in the *application.yaml* and configure the default fuzziness in the range [0.0, 1.0]. The value 1.0 enforces exact search. The default fuzziness is 0.8.
```
Comment (renejeglinsky) [inline guides/providing-services.md:423]: How is that configured in Node.js? We could as well profile the sentence and add what's relevant for Node.js here.

Comment (johannes-vogel) [reply guides/providing-services.md:423]: We do not have an hex optimization mode. hex engine is always used.

Comment (renejeglinsky) [reply guides/providing-services.md:423]:
```suggestion
If you run in [`HEX` optimization mode (default for Node.js)](../java/cqn-services/persistence-services#sql-optimization-mode) on SAP HANA Cloud, you can enable fuzzy search and configure the default fuzziness in the range [0.0, 1.0]. The value 1.0 enforces exact search. The default fuzziness is 0.8.

- Java: <Config java>cds.sql.hana.search.fuzzy = true</Config>
- Java: <Config java>cds.sql.hana.search.fuzzinessThreshold = 0.9</Config>
- Node.js:<Config>cds.hana.fuzzy = 0.9</Config>
```

File: guides/providing-services.md [hunk 2]
```diff
@@ -420,14 +420,30 @@ You can explicitly annotate calculated elements to make them searchable, even th
...
+```json
+"cds": {
+  "hana": {
+    "fuzzy": 0.9
+  }
+}
+```
+
+</span>
+
```
Comment (renejeglinsky) [inline guides/providing-services.md:446]:
```suggestion

```

File: guides/providing-services.md [hunk 3]
```diff
@@ -420,14 +420,30 @@ You can explicitly annotate calculated elements to make them searchable, even th
...
 Override the fuzziness for elements, using the `@Search.fuzzinessThreshold` annotation:
```
Comment (renejeglinsky) [inline guides/providing-services.md:447]: Is this annotation any different for Node.js?

File: guides/providing-services.md [hunk 4]
```diff
@@ -454,8 +470,8 @@ When using wildcards in search terms, an *exact pattern search* is performed.
 Supported wildcards are '*' matching zero or more characters and '?' matching a single character. You can escape wildcards using '\\'.
 :::
 
-::: warning Only Java
-Fuzzy search on SAP HANA Cloud is currently only supported on the CAP Java runtime and requires the [`HEX` optimization mode](../java/cqn-services/persistence-services#sql-optimization-mode).
+::: warning
+Fuzzy search on SAP HANA Cloud requires the [`HEX` optimization mode](../java/cqn-services/persistence-services#sql-optimization-mode).
```
Comment (renejeglinsky) [inline guides/providing-services.md:474]:
```suggestion
Fuzzy search on SAP HANA Cloud requires the [`HEX` optimization mode (default for Node.js)](../java/cqn-services/persistence-services#sql-optimization-mode).
```

File: guides/providing-services.md (hunk 1 above)
Comment (renejeglinsky) [reply guides/providing-services.md:423]: The default fuzziness is different in Node.js, isn't it? It's 0.7 and not 0.8, right?

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**